internal/sbi/processor/pdu_session.go

```diff
@@ -121,6 +121,17 @@ func (p *Processor) HandlePDUSessionSMContextCreate(
 		}
 	}
 
+	if !p.Context().Ues.UeExists(smContext.Supi) {
```

Comment:
What if the error happens after the SDM data changed?
I think use the `defer` would be better in this case:
```go
defer func(err error) {
  if err != nil {
	if !p.Context().Ues.UeExists(smContext.Supi) {
		if problemDetails, err := p.Consumer().
			Subscribe(ctx, smContext, smPlmnID); problemDetails != nil {
			smContext.Log.Errorln("SDM Subscription Failed Problem:", problemDetails)
		} else if err != nil {
			smContext.Log.Errorln("SDM Subscription Error:", err)
		}
	} else {
		p.Context().Ues.IncrementPduSessionCount(smContext.Supi)
	}
  }
}(err)
```